recipes/coremltools/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=10.13 \
```

Comment:
Doesn't this work with 10.9?

Reply:
I don't believe so: https://github.com/conda-forge/staged-recipes/pull/12900#issuecomment-709388849. It uses headers that are only available in 10.13+ (I think).